packages/mui-material/src/Paper/Paper.js

```diff
@@ -5,21 +5,11 @@ import { chainPropTypes, integerPropType } from '@mui/utils';
 import { unstable_composeClasses as composeClasses } from '@mui/base';
 import { alpha } from '@mui/system';
 import styled from '../styles/styled';
+import getOverlayAlpha from '../styles/getOverlayAlpha';
 import useThemeProps from '../styles/useThemeProps';
 import useTheme from '../styles/useTheme';
 import { getPaperUtilityClass } from './paperClasses';
 
-// Inspired by https://github.com/material-components/material-components-ios/blob/bca36107405594d5b7b16265a5b0ed698f85a5ee/components/Elevation/src/UIColor%2BMaterialElevation.m#L61
-export const getOverlayAlpha = (elevation) => {
```

Comment:
Needed to be used in the `extendTheme` in `@mui/material-next` so it was extracted and exported.